website/docs/r/service_account.html.markdown

```diff
@@ -76,4 +76,4 @@ The following arguments are supported:
 In addition to the arguments listed above, the following computed attributes are
 exported:
 
-* `default_secret_name` - Name of the default secret the is created & managed by the service
+* `default_secret_name` - Name of the default secret created & managed by the service. If there is more than one secret, the first with "token" in name is selected or, if none, the first in list.
```

Comment:
To ensure the last part of that sentence, the assignment at https://github.com/terraform-providers/terraform-provider-kubernetes/pull/281/files#diff-375ac94c5bd4e304ddeb13226700ec77R112 needs to be moved out of the condition block.

Reply:
Oops, missed https://github.com/terraform-providers/terraform-provider-kubernetes/pull/281/files#diff-375ac94c5bd4e304ddeb13226700ec77R108

It's ok then.